capture/field.c

```diff
@@ -1382,6 +1384,14 @@ void arkime_field_free(ArkimeSession_t *session)
             }
             ARKIME_TYPE_FREE(ArkimeCertsInfoHashStd_t, cihash);
             break;
+        case ARKIME_FIELD_TYPE_OBJECT:
+            ArkimeFieldObjectFreeFunc freeCB = config.fields[pos]->object_free;
```

Comment:
this is failing on old compilers. you need to have { } after the case and before the break, or move the freeCB variable declaration above the switch. I do both. :)

Reply:
OK, did not know that, I'll fix it when I get in front of my computer
Thanks